notify/impl.go

```diff
@@ -584,7 +584,7 @@ func (n *Hipchat) Notify(ctx context.Context, as ...*types.Alert) error {
 	}
 
 	req := &hipchatReq{
-		From:          tmplText(n.conf.From),
+		From:          truncate(tmplText(n.conf.From), 64),
```

Comment:
It's in the spec: https://www.hipchat.com/docs/apiv2/method/send_room_notification